packages/mgt/src/graph/graph.photos.ts

```diff
@@ -69,37 +69,48 @@ export function myPhoto(graph: IGraph): Promise<string> {
  */
 export async function getPersonImage(graph: IGraph, person: IDynamicPerson) {
   let image: string;
+  let email: string;
 
   if ((person as MicrosoftGraph.Person).userPrincipalName) {
     // try to find a user by userPrincipalName
     const userPrincipalName = (person as MicrosoftGraph.Person).userPrincipalName;
     image = await getUserPhoto(graph, userPrincipalName);
-  } else {
-    if (person.id) {
-      image = await getUserPhoto(graph, person.id);
-      if (image) {
-        return image;
-      }
+  } else if ((person as any).personType.subclass === 'PersonalContact') {
+    // if person is a contact, look for them and their photo in contact api
+    email = getEmailFromGraphEntity(person);
+    const contact = await findContactByEmail(graph, email);
+    if (contact[0] && contact[0].id) {
```

Comment:
`findContactByEmail` can return a null value in some cases.

This small tweak should fix it:
`if (contact && contact.length && contact[0].id) {`